TheMovieDb/Home/Contoller/HomeViewController.swift

```diff
@@ -7,21 +7,222 @@
 
 import UIKit
 
-final class HomeViewController: UIViewController {
-    
+final class HomeViewController: UICollectionViewController {
     // MARK: - Properties
+    private var topRatedMovies = [Movie]()
+    private var playingNowMovies = [Movie]()
+    private var upComingMovies = [Movie]()
+    private var trendingMovies = [Movie]()
+    private var popularMovies = [Movie]()
+    private let group = DispatchGroup()
     
     // MARK: - Life Cycle
+    
+    init() {
+        super.init(collectionViewLayout: UICollectionViewController.configureCollectionViewLayout())
```

Comment:
nice to move the creation somewhere else

Reply:
fixed

Reply:
I moved it where the controller is instance